### PR TITLE
Add missing efs and msk to serverless yml md

### PR DESCRIPTION
**Pull request by pgrzesik**
While adding `useDotenv` to `serverless.yml.md` file, I realised that I didn't update it while implementing support for msk and efs - that PR should fix that.

### CODE REVIEW

**Review by medikoo — APPROVED** [review]
Thank you @pgrzesik (I also tend to forget to request that, great you've noticed!)

File: docs/providers/aws/guide/serverless.yml.md [hunk 1]
```diff
@@ -365,6 +368,12 @@ functions:
           maximumRecordAgeInSeconds: 120
           startingPosition: LATEST
           enabled: false
+      - msk:
+          arn: arn:aws:kafka:us-east-1:111111111111:cluster/ClusterName/a1a1a1a1a1a1a1a1a
+          topic: kafkaTopic
+          batchSize: 100
+          startingPosition: LATEST
+          enabled: false
```
Comment (medikoo) [inline docs/providers/aws/guide/serverless.yml.md:376]: When there are defaults, let's use _default_ values (so `true` in this case). Also it's nice to note in comment on optional properties

**Review by medikoo — CHANGES_REQUESTED** [review]
@pgrzesik sorry, at last call I've noticed that as proposed it doesn't meet our conventions. Please see my comment

File: docs/providers/aws/guide/serverless.yml.md (hunk 1 above)
Comment (pgrzesik) [reply docs/providers/aws/guide/serverless.yml.md:376]: Thanks for catching that - I followed the same convention that I saw in `stream` event right above it - should it also have `enabled` set to `true` here?

Comment (medikoo) [reply docs/providers/aws/guide/serverless.yml.md:376]: > should it also have enabled set to `true` here?

Yes, definitely, somehow it was missed

**Review by medikoo — APPROVED** [review]
Thank you @pgrzesik !

**Pull request merged**